dali/daliadmin/daliadmin.cpp

```diff
@@ -269,11 +270,13 @@ int main(int argc, const char* argv[])
                     }
                     else if (strieq(cmd,"set")) {
                         CHECKPARAMS(2,2);
-                        set(params.item(1),params.item(2));
+                        setValue(params.item(1),params.item(2),out);
+                        PROGLOG("%s",out.str());
```

Comment:
as noted in daadmin.cpp setValues, woud be better if setValues just returned old value, and the output message was here